content/includes/business-logos.html

```diff
@@ -0,0 +1,19 @@
+<section id="business-logo">
+    <div class="ui very padded container">
+        <div class="ui four column grid">
+            <div class="column">
+                <img class="ui small image" src="{{ SITE_URL }}/images/amd-logo.svg" alt="AMD Logo">
+            </div>
+            <div class="column">
+                <img class="ui small image" src="{{ SITE_URL }}/images/canonical-logo.svg" alt="Canonical Logo">
+            </div>
+            <div class="column">
+                {# This is medium because the logo is small otherwise? :shrug: #}
```

Comment:
This is mainly because the images are all of varying dimensions. It's best to edit the images down to a consistent dimension for this.

Also note that `.ui.small.image` scales the image to the _width_ dimension, which is why it looks especially small in comparison